source/_integrations/light.mqtt.markdown

```diff
@@ -15,15 +15,17 @@ The `mqtt` light platform lets you control your MQTT enabled lights through one
+| Color mode        | ✔                                                          | ✔                                                                    | ✘                                                                              |
 | Color temperature | ✔                                                          | ✔                                                                    | ✔                                                                            |
 | Effects           | ✔                                                          | ✔                                                                    | ✔                                                                            |
 | Flashing          | ✘                                                          | ✔                                                                    | ✔                                                                            |
+| HS Color          | ✔                                                          | ✔                                                                    | ✘                                                                            |
 | RGB Color         | ✔                                                          | ✔                                                                    | ✔                                                                            |
+| RGBW Color        | ✔                                                          | ✔                                                                    | ✘                                                                            |
+| RGBWW Color       | ✔                                                          | ✔                                                                    | ✘                                                                            |
 | Transitions       | ✘                                                          | ✔                                                                    | ✔                                                                            |
-| XY Color          | ✔                                                          | ✔                                                                    | ✘                                                                            |
-| HS Color          | ✔                                                          | ✔                                                                    | ✘                                                                            |
 | White Value       | ✔                                                          | ✔                                                                    | ✔                                                                            |
```

Comment:
Are we going to deprecate white value? Should we mention somewhere the relationship between white value and white?

Reply:
We could just remove it instead of documenting deprecated features?

Reply:
That is the general approach in end-user documentation. Removing deprecated documentation, which in general, works pretty well.